subsys/net/openthread/Kconfig

```diff
@@ -64,7 +65,7 @@ config OPENTHREAD_NRF_SECURITY_PSA
 
 config OPENTHREAD_NRF_SECURITY_PSA
 	bool
-	select MBEDTLS_PSA_CRYPTO_STORAGE_C
+	select MBEDTLS_PSA_CRYPTO_STORAGE_C if !PSA_SSF_CRYPTO_CLIENT
 	select TRUSTED_STORAGE
```

Comment:
not related much.. but should we enable Trusted Storage for TF-M or `PSA_SSF_CRYPTO_CLIENT` (nRF54H)? Not sure if we use `OPENTHREAD_NRF_SECURITY_PSA` anywhere in Matter.